code/modules/antagonists/voidwalker/voidwalker_abilities.dm

```diff
@@ -0,0 +1,113 @@
+/// Remain in someones view without moving yourself to stun and weaken them for a bit
```

Comment:
but in the OP, you said "Both you and your target can move, as long as you remain in their view."